AdAway/src/main/res/values/strings.xml

```diff
@@ -222,5 +224,9 @@
     <string name="pref_enable_debug_summary">You need to restart AdAway for this setting to take effect.</string>
     <string name="pref_enable_ipv6">Enable IPv6</string>
     <string name="pref_enable_ipv6_summary">All host file entries will get a corresponding ::1 (IPv6 localhost) entry.</string>
+    <string name="pref_enable_systemless_def">false</string>
+    <string name="pref_enable_systemless_key">enableSystemless</string>
+    <string name="pref_enable_systemless_summary"><![CDATA[Prevent AdAway to modify system partition (requires SuperSU >= 2.76)]]></string>
```

Comment:
"Prevent AdAway **to modify** system partition" -> "Prevent AdAway **from modifying the** system partition"